### PR TITLE
Add description for contribution rates of social insurances

### PR DESCRIPTION
**Pull request by paulinaschroeder**
### What problem do you want to solve?

Add description for contribution rates of social insurances. Without checking it was previously unclear if those parameters are the combined rates of employers and employees or just the rate for employees. 

### Todo

- [x] add description for care insurance
- [x] add description for unemployment insurance
- [x] add description for care pension insurance

### CODE REVIEW

File: gettsim/parameters/soz_vers_beitr.yaml [hunk 1]
```diff
@@ -10,13 +10,26 @@ beitr_satz:
                     sonderbeitrag - bezahlt von Arbeitnehmern, ab Jul 2005 bis 2014
                     ermäßigt - ermäßigter Satz (§243 SGB V)
                     zusatz - durchschnittlicher Zusatzbeitragssatz.
+      ges_pflegev: Beitragssätze für gesetzliche Pflegeversicherung.
+                    standard - Beitragssatz für Arbeitnehmer
```
Comment (ChristianZimpelmann) [inline gettsim/parameters/soz_vers_beitr.yaml:14]:
```suggestion
                   standard - Beitragssatz für Arbeitnehmer
```
also the next lines

File: gettsim/parameters/soz_vers_beitr.yaml [hunk 2]
```diff
@@ -10,13 +10,26 @@ beitr_satz:
...
+                    zusatz_kinderlos - Beitragszuschlag für kinderlose Versicherte,
+                    die das 23. Lebensjahr vollendet haben. Wird vom Arbeitnehmer
+                    alleine getragen. Eingeführt 2005.
+      arbeitsl_v:  Beitragssätze zur Arbeitslosenversicherung für Arbeitnehmer.
+      ges_rentenv: Beitragssätze zur allgemeinen Rentenversicherung für Arbeitnehmer.
     en:
-      ges_krankenv: Statury Contribution Rates for mandatory health insurance.
+     ges_krankenv: Statury Contribution Rates for mandatory health insurance.
```
Comment (ChristianZimpelmann) [inline gettsim/parameters/soz_vers_beitr.yaml:21]:
```suggestion
      ges_krankenv: Statury Contribution Rates for mandatory health insurance.
```
Here we need an additional whitespace (two more as in the intendation level before)

File: gettsim/parameters/soz_vers_beitr.yaml [hunk 3]
```diff
@@ -10,13 +10,26 @@ beitr_satz:
...
                     mean_allgemein - average contribution rate, 1992 to 2008
                     allgemein - general statutory contribution rate, since 2009
                     sonderbeitrag - paid by employees, Jul 2005 to 2014
                     ermäßigt - reduced rate
                     zusatz - average top-up contribution rate
+     ges_pflegev: 	Statury Contribution Rates for mandatory care insurance.
```
Comment (ChristianZimpelmann) [inline gettsim/parameters/soz_vers_beitr.yaml:27]: Too many whitespaces before "Statury"

**Review by ChristianZimpelmann — CHANGES_REQUESTED** [review]
Nice improvement! Thanks!

Just the intendation doesn't seem to fit always (I made a few comments). Did you use pre-commit hooks? Then some of the those should be fixed automatically.

File: gettsim/parameters/soz_vers_beitr.yaml (hunk 3 above)
Comment (paulinaschroeder) [reply gettsim/parameters/soz_vers_beitr.yaml]: Everything should now line up as intended

**Review by ChristianZimpelmann — APPROVED** [review]
Great! Thanks!

**Pull request merged**